### DISCUSSION TITLE
版本23.05.2固件使用中发生故障。

### DISCUSSION THREAD

**Discussion by lysgwl** [immortalwrt/immortalwrt#1323] [Q&A]
### Describe the bug

设备r2s编译的Immortalwrt23.05.2固件，运行大概一天多，luci界面无法登录，显示以下信息：
No root node was registered, this usually happens if no module was installed.
Install luci-mod-admin-full and retry. If the module is already installed, try removing the /tmp/luci-indexcache file.

登录ssh后台，任何命令操作后，也发生错误：-ash: ls: I/O error
网络运行正常，luci web界面和ssh无法操作。

### ImmortalWrt version

23.05.2

### ImmortalWrt release

r27625

### ImmortalWrt target/subtarget

416c8c5c91

### Device

r2s

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by nicholas-opensource** [comment]
你好lysgwl，这是开发板的常见问题，通常是TF卡接近写入保护了，TLC的卡前端磨损过大（kernelfs，rootfs区域）。或者高温/接触不良。

**Comment by 1715173329** [comment]
_Troubleshooting any problem without the error log is like driving with your eyes closed.
  -- [Apache Getting Started](https://httpd.apache.org/docs/2.4/getting-started.html)_

**Comment by Arjen10** [comment]
加风扇，我是高温，上拼多多买了两个风扇装上就不烫手了。自然也就正常了
